shinyswatch/_get_theme_deps.py

```diff
@@ -33,40 +68,61 @@ def get_theme_deps(name: BSW5_THEME_NAME) -> list[HTMLDependency]:
         # This is to prevent the Shiny bootstrap stylesheet from being loaded and instead load the bootswatch + bootstrap stylesheet
         # _Disable_ bootstrap html dep
         # Prevents  bootstrap from being loaded at a later time (Ex: shiny.ui.card() https://github.com/rstudio/py-shiny/blob/d08af1a8534677c7026b60559cd5eafc5f6608d7/shiny/ui/_navs.py#L983)
+        *suppress_shiny_bootstrap(),
         HTMLDependency(
-            name="bootstrap",
-            version=base_dep_version,
-        ),
-        # Use a custom version of bootstrap with no stylesheets/JS
-        bs_dep_no_files,
-        # Add in the matching JS files
-        HTMLDependency(
-            name="bootstrap-js",
+            name="shinyswatch-js",
```

Comment:
Can be replaced with `dep_shinyswatch_bootstrap_js()`?

Reply:
Oops, yeah I meant to do that when I refactored.